docs/v-secure/guides/mcp-integration.md

```
@@ -0,0 +1,438 @@
+### Requesting Secret Access
+
+In Claude, you can now request access to secrets:
+
+```
+Claude, I need to debug the database connection.
+Can you read the DATABASE_URL secret?
+```
+
+Claude will use the MCP protocol to request approval:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)